src/containers/BookForm.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-tabs, jsx-a11y/label-has-for */
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React from 'react';
+
+const BookForm = () => (
+  <div>
+    <form>
+      <div>
+        <label htmlFor="title">
+					Book Title:
+          <input type="text" placeHolder="enter book title" name="title" id="title" />
+        </label>
+      </div>
+      <div>
+        <label> Book Category:</label>
+        <select>
+          <option value="Action"> Action </option>
+          <option value="Biography"> Biography </option>
+          <option value="History"> History</option>
+          <option value="Horro"> Horror</option>
+          <option value="Kids"> Kids </option>
+          <option value="Learning"> Learning </option>
+          <option value="Sci-Fi"> Sci-Fi </option>
+        </select>
```

Comment:
Like the `title` input, we could wrap the `select` element with the `label` tag and define the `htmlFor` and `id` for the `label` and `select` tag respectively. This would help with the two rules disabled `/* eslint-disable no-tabs, jsx-a11y/label-has-for */` and `/* eslint-disable jsx-a11y/label-has-associated-control */` and also make the form more accesible.